### PR TITLE
fix(quay): Format date, size and Manifest information to be same as on the quay UI

### PR DESCRIPTION
**Pull request by fmenesesg**
#100

 Format date, size and Manifest information to be same as on the quay UI

### CODE REVIEW

**Review by tumido — CHANGES_REQUESTED** [review]
There seems to be some formatting errors in here. Can you try running `prettier` please?

**Comment by fmenesesg** [comment]
I execute prettier, now should be ok, can you PTAL

File: plugins/quay/src/components/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+export function formatSize(sizeInBytes: number) {
+  if (!sizeInBytes) {
+    // null or undefined
+    return 'N/A';
+  }
+
+  const i = Math.floor(Math.log(sizeInBytes) / Math.log(1024));
+  return `${Number((sizeInBytes / Math.pow(1024, i)).toFixed(2)) * 1} ${
```
Comment (tumido) [inline plugins/quay/src/components/utils.tsx:40]: Hm, can you explain me what is the `* 1` magic about? I'm unable to find a functional difference.

```ts
  return `${Number((sizeInBytes / Math.pow(1024, i)).toFixed(2)) * 1} ${
                                                                 ^^^^
```

File: plugins/quay/src/components/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    ['B', 'kB', 'MB', 'GB', 'TB'][i]
```
Comment (tumido) [inline plugins/quay/src/components/utils.tsx:41]: Wouldn't it be better if this array was a global constant? I may be too picky here, just a nit.

File: plugins/quay/src/components/utils.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    case VulnerabilitySeverity.Medium:
+      return '#EC7A08';
+    case VulnerabilitySeverity.Low:
+      return '#F0AB00';
+    case VulnerabilitySeverity.None:
+      return '#3E8635';
+    default:
+      return '#8A8D90';
+  }
+}
```
Comment (tumido) [inline plugins/quay/src/components/utils.tsx:18]: I think a constant would be nice here, but I may be too picky:

```suggestion
export function getSeverityColor(severity: VulnerabilitySeverity) {
export const SEVERITY_COLORS = new Proxy(
  {
    VulnerabilitySeverity.Critical: '#7D1007',
    VulnerabilitySeverity.High: '#C9190B',
    VulnerabilitySeverity.Medium: '#EC7A08',
    VulnerabilitySeverity.Low: '#F0AB00',
    VulnerabilitySeverity.None: '#3E8635'
  },
  {
    get: (o, k) => k in o ? o[k] : '#8A8D90'
  }
)
```

This should be easily addressable via `SEVERITY_COLORS[severity]`.


Again, just a nit, if you don't think agree, feel free to leave as it is, your implementation is completely reasonable and understandable.

**Review by tumido — COMMENTED** [review]
LGTM overall, I've left couple of suggestions. Feel free to dismiss them, this is mergeable as is.

**Review by tumido — APPROVED** [review]
After DM discussion the nits are filed as a separate refactor issue. It's minor changes anyways... We can merge this as is. LGTM :+1:

**Pull request merged**